### PR TITLE
updated config for slider facet to use constants

### PR DESCRIPTION
**Pull request by huuaho**
Relies on Package update: https://github.com/CBIIT/bento-frontend/pull/904

The sliders have an updated config to use the declared constants defined in the packages.

### CODE REVIEW

**Comment by huuaho** [comment]
Let's not merge this PR until we have the package publish and merge this PR with the package.json update as well

**Review by iksheth — APPROVED** [review]

**Pull request merged**
